manifests/plugin/powerdns.pp

```diff
@@ -0,0 +1,56 @@
+# See http://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_powerdns
+class collectd::plugin::powerdns (
+  $ensure                                 = 'present',
+  $order                                  = 10,
+  $interval                               = undef,
+  Optional[Hash[String, Hash]] $servers   = undef,
+  Optional[Hash[String, Hash]] $recursors = undef,
+  Optional[String] $local_socket          = undef,
+) {
+
+  include ::collectd
```

Comment:
please don't add the :: in front of classes

Reply:
done